composer.json

```diff
@@ -20,9 +20,8 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.7",
-        "doctrine/inflector": "1.0.*",
-        "knplabs/knp-paginator-bundle": "2.4.*"
+        "symfony/framework-bundle": "~3.1",
```

Comment:
Do we stop ensuring support for versions lower than `3.1.0`? 
Otherwise it should be `~2.7|~3.0`.